pkg/virt-controller/services/template.go

```diff
@@ -539,6 +540,29 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, i
 			TopologySpreadConstraints:     vmi.Spec.TopologySpreadConstraints,
 		},
 	}
+	tscFrequency := nsr.TSCFrequency()
+	if tscFrequency != 0 {
+		pod.Spec.Affinity = &k8sv1.Affinity{
+			NodeAffinity: &k8sv1.NodeAffinity{
+				RequiredDuringSchedulingIgnoredDuringExecution: &k8sv1.NodeSelector{
+					NodeSelectorTerms: []k8sv1.NodeSelectorTerm{{
+						MatchExpressions: []k8sv1.NodeSelectorRequirement{
+							{
+								Key:      topology.TSCFrequencyLabel,
+								Operator: "Gt",
+								Values:   []string{strconv.FormatInt(tscFrequency-533000, 10)},
```

Comment:
Make `533000` a constant with a comment where the magic number comes from?

Reply:
Thanks for the review. This is actually incorrect, the exact value should be calculated as 250ppm of the frequency. Fixing soon.
/hold